sycl/include/CL/sycl/info/info_desc.hpp

```diff
@@ -287,7 +287,10 @@ enum class event : cl_event_info {
 enum class event_command_status : cl_int {
   submitted = CL_SUBMITTED,
   running = CL_RUNNING,
-  complete = CL_COMPLETE
+  complete = CL_COMPLETE,
+  ext_oneapi_unknown =
+      submitted + running + complete +
+      1 // TODO: a more elegant way to ensure that the unique value is here
```

Comment:
It would leave enough space for 1:1 mapping if BE supports more statuses.
Maybe @gmlueck have opinion as well.

Reply:
You are worried that the value of `ext_oneapi_unknown` might collide with some future OpenCL extension that adds a new "command execution status" value?  It's hard to predict what new values might be added in the future.  However, since all values are positive now, it's probably safe to use a negative value.  Therefore, you could use `-1`.  If we add other `ext_oneapi` values in the future, they could be `-2`, etc.

Reply:
Ok, I changed it to `-1`, thanks for the suggestion!